packages/@o3r/core/schematics/service/index.spec.ts

```diff
@@ -38,6 +38,21 @@ describe('Service generator', () => {
       path: 'src/services'
     }, initialTree);
 
+    expect(tree.files.filter((file) => /test-service/.test(file)).length).toEqual(9);
+    expect(tree.files.some((file) => /^[\\/]?src[\\/]services[\\/]test-service[\\/]test-base[\\/]test-service\.test-base\.module\.ts$/i.test(file))).toBeTruthy();
+  });
+
+  // TODO enable when https://github.com/jestjs/jest/issues/9543 fixed
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('should generate service with fixtures for jest if installed', async () => {
+    const runner = new SchematicTestRunner('schematics', collectionPath);
+    const tree = await runner.runExternalSchematic('schematics', 'service', {
+      projectName: 'test-project',
+      name: 'test-service',
+      featureName: 'test-base',
+      path: 'src/services'
+    }, initialTree);
+
     expect(tree.files.filter((file) => /test-service/.test(file)).length).toEqual(12);
     expect(tree.files.some((file) => /^[\\/]?src[\\/]services[\\/]test-service[\\/]test-base[\\/]test-service\.test-base\.module\.ts$/i.test(file))).toBeTruthy();
```

Comment:
expectation is not the good one I think

Reply:
when the test will be re-enabled it will be the proof that it works correctly (we will have to adapt the no of files)